packages/main/src/plugin/image-registry.ts

```diff
@@ -159,4 +169,71 @@ export class ImageRegistry {
     this.apiSender.send('registry-update', registry);
     this._onDidUpdateRegistry.fire(Object.freeze(registry));
   }
+
+  async getAuthUrl(serviceUrl: string): Promise<string | undefined> {
+    let registryUrl: string;
+
+    if (serviceUrl.includes('docker.io')) {
+      registryUrl = 'https://index.docker.io/v2/';
+    } else {
+      registryUrl = serviceUrl + '/v2/';
+
+      if (!registryUrl.startsWith('http')) {
+        registryUrl = 'https://' + registryUrl;
+      }
+    }
+
+    try {
+      await got.get(registryUrl);
+    } catch (requestErr) {
+      if (requestErr instanceof HTTPError && requestErr.response?.statusCode === 401) {
+        console.log('Registry ' + registryUrl + ' requires authentication');
```

Comment:
do we need these traces ?

Reply:
these traces needs only for the debug purposes, should we keep them or are they redundant?